scripts/terminalserver/terminalserver.jl

```diff
@@ -358,32 +377,46 @@ Base.Multimedia.istextmime(::MIME{Symbol("application/vnd.dataresource+json")})
 
 displayable(d::InlineDisplay, ::MIME{Symbol("application/vnd.plotly.v1+json")}) = true
 
+const DISPLAYABLE_MIMES = [
+    "application/vnd.vegalite.v4+json",
+    "application/vnd.vegalite.v3+json",
+    "application/vnd.vegalite.v2+json",
+    "application/vnd.vega.v5+json",
+    "application/vnd.vega.v4+json",
+    "application/vnd.vega.v3+json",
+    "application/vnd.plotly.v1+json",
+    "juliavscode/html",
+    # "text/html",
+    "image/svg+xml",
+    "image/png"
+]
+
+function can_display(x)
+    for mime in DISPLAYABLE_MIMES
+        if showable(mime, x)
+            return true
+        end
+    end
+
+    if showable("application/vnd.dataresource+json", x)
+        return true
+    end
+
+    if _isiterabletable(x) === missing || _isiterabletable(x) === true || x isa AbstractVector || x isa AbstractMatrix
```

Comment:
```suggestion
    istable = Base.invokelatest(_isiterabletable, x)

    if istable === missing || istable === true || x isa AbstractVector || x isa AbstractMatrix
```
I just got a world age error without this.